spec/system/claims/schools/mentors/claims_school_user_adds_mentor_to_school_spec.rb

```diff
@@ -0,0 +1,267 @@
+require "rails_helper"
+
+RSpec.describe "Claims school user adds mentors to schools", service: :claims, type: :system do
+  scenario do
+    given_an_eligible_school_exists_with_an_assigned_mentor_and_an_unassigned_one
+    and_given_the_get_teacher_api_is_stubbed
+    and_i_am_signed_in
+
+    when_i_navigate_to_mentors
+    then_i_see_the_mentors_index_page
```

Comment:
```suggestion
    given_an_eligible_school_exists_with_an_assigned_mentor_and_an_unassigned_one
    and_given_the_get_teacher_api_is_stubbed
    and_i_am_signed_in
    and_i_navigate_to_mentors
    then_i_see_the_mentors_index_page
```

We could test that we see the root path after signing in as well, but I'll accept this.